src/app/cli/src/commands/sql_shell_command.rs

```diff
@@ -124,17 +129,30 @@ impl SqlShellCommand {
 
         Ok(())
     }
+
+    async fn run_datafusion_cli_command(&self) -> Result<(), CLIError> {
+        let mut print_options = PrintOptions {
+            format: PrintFormat::Table,
+            quiet: true,
```

Comment:
Do you know what this option does? 

I'd suspect that quiet mode is used for executing scripts not to produse some verbose messages like table creation confirmations etc.

Perhaps in interactive shell mode we should be verbose.

Reply:
taken from   https://arrow.apache.org/datafusion/user-guide/cli.html:  -q, --quiet
            Reduce printing other than the results and work quietly

Reply:
I don't mind changing it to false then we have more info.

Reply:
I tested locally and all seems to work as expected:
- I can query things
- `join` things
- `explain` queries
- and I get helpful error messages on invalid schema and query mistakes 

I tried with `quiet: true/false` and didn't notice any difference.